.gitlab-ci.yml

```diff
@@ -0,0 +1,55 @@
+include:
+  - project: 'city-of-helsinki/kuva/ci-cd-config/ci-configuration'
+    ref: v1
+    file: '/.gitlab-ci-template.yml'
+
+variables:
+  SERVICE_PORT: 80
+  DOCKER_BUILD_ARG_REACT_APP_OIDC_CLIENT_ID: "https://api.hel.fi/auth/kukkuu-ui"
```

Comment:
I would move this to build jobs' variables even if that means repeating the same value three times.
But that's just a matter of taste.

Reply:
I realised that we don't need this as a changeable environment variable, so I just removed it. 

Will keep your suggestion in mind (and only not consider it if the list of variables is long..)